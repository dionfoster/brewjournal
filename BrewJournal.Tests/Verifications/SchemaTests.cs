using System.Data.Entity.Infrastructure;
using ApprovalTests;
using ApprovalTests.Reporters;
using BrewJournal.Tests.Testability;
using Xunit;

namespace BrewJournal.Tests.Verifications
{
    public class SchemaTests : DatabaseFixture
    {
        [Fact]
        [UseReporter(typeof(DiffReporter))]
        public void VerifySchemaChange()
        {
            var sqlScript = ((IObjectContextAdapter)WorkDbContext).ObjectContext.CreateDatabaseScript();

            Approvals.Verify(sqlScript);
        }
    }
}