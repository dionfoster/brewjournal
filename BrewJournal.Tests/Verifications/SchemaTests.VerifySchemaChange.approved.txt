create table [dbo].[Brews] (
    [Id] [uniqueidentifier] not null default newid(),
    [Name] [nvarchar](255) not null,
    primary key ([Id])
);
